v2/pkg/reporting/format/format.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/projectdiscovery/nuclei/v2/pkg/types"
 )
 
+const builderEndWithAcutes = "\n```\n"
```

Comment:
Instead of extracting this as a constant, we should probably rather have a method methods like:

```go
func createMarkdownCodeBlock(title string, content string, language string) string {
	return "\n" + createBoldMarkdown(title) + "\n```" + language + "\n" + content + "\n```\n"
}

func createBoldMarkdown(value string) string {
	return "**" + value + "**"
}
```

Reply:
Used suggested methods instead of extracting it as a constant